minitests/iostandard/features/generate.py

```diff
@@ -0,0 +1,491 @@
+import os
+import random
+import json
+from collections import defaultdict
+
+try:
+    random.seed(int(os.getenv("SEED"), 16))
+except TypeError:
+    pass
+
+# =============================================================================
+
+
+def load_iob_sites(file_name):
+    """
+    Loads IOB site dump from the given CSV file.
+    """
+    iob_sites = defaultdict(lambda: [])
+
+    with open(file_name, "r") as fp:
+        fp.readline()
+
+        for line in fp:
```

Comment:
Just use `csv.DictReader`?  More flexible in the event of more columns, and less code, and it does the same thing?

Reply:
Right, switched to `csv.DictReader`